### PR TITLE
Graph protocol chart

### PR DESCRIPTION
**Pull request by JacqueGM**
- graph chart
- graph chart upgrade script
- subgraph deployments k8s Jobs

### CODE REVIEW

**Comment by JacqueGM** [comment]
Closes https://github.com/CirclesUBI/circles-iac/issues

**Review by llunaCreixent — APPROVED** [review]
Looks good!
the new env  var is there: https://github.com/CirclesUBI/circles-subgraph/blob/44-add-setupSafe-event/.env.example#L21
And the value comes from: https://github.com/safe-global/safe-deployments/blob/main/src/assets/v1.3.0/proxy_factory.json

File: helm/circles-graphprotocol/values.yaml [hunk 1]
```diff
@@ -45,4 +45,15 @@ acmeServer: https://acme-v02.api.letsencrypt.org/directory
 # ethereum networks:
 network: xdai
 rpcNode: xdai-nethermind.xdai
-rpcPort: 8545
\ No newline at end of file
+rpcPort: 8545
+
+# CORS entried allowed: 
+CORS_allowed:
+  - "https://relayr.circles.garden"
```
Comment (llunaCreixent) [inline helm/circles-graphprotocol/values.yaml:52]: typo?

Comment (llunaCreixent) [reply helm/circles-graphprotocol/values.yaml:52]: Also this list should be reviewed

**Review by llunaCreixent — CHANGES_REQUESTED** [review]

File: helm/circles-subgraphs/subgraph-job-130.yaml [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+                name: env
+        - name: postgres_host
+          value: postgres
+        - name: postgres_user
+          value: graph-node
+        - name: postgres_pass
+          value: let-me-in
+        - name: postgres_db
+          value: graph-node
+        image: registry.digitalocean.com/circles-registry/circles-subgraph:v1.3.3-3
```
Comment (llunaCreixent) [inline helm/circles-subgraphs/subgraph-job-130.yaml:59]: I'm not sure which is the exact version

Comment (JacqueGM) [reply helm/circles-subgraphs/subgraph-job-130.yaml:59]: this is the latest version from https://github.com/CirclesUBI/circles-subgraph/actions/runs/3481090186

File: helm/circles-graphprotocol/values.yaml (hunk 1 above)
Comment (JacqueGM) [reply helm/circles-graphprotocol/values.yaml]: yes, if you tell me which endpoint remove and add will change them

Comment (JacqueGM) [reply helm/circles-graphprotocol/values.yaml]: fixed typo

File: ansible/roles/graph-protocol/files/values-prd.yaml [hunk 3]
```diff
@@ -1,12 +1,7 @@
 # ethereum network
-network: xdai
+ethereumNetwork: xdai
 rpcNode: xdai-nethermind.xdai
-# subgraph values
-subgraph_name: circlesubi/circles-ubi
-subgraph_network: xdai
-proxy_factory_address: "0x8b4404DE0CaECE4b966a9959f134f0eFDa636156"
-start_block: "12529458"
-hub_address: "0x29b9a7fBb8995b2423a71cC17cf9810798F6C543"
```
Comment (llunaCreixent) [inline ansible/roles/graph-protocol/files/values-prd.yaml:9]: Why don't we maintain these config values? They are reused in both sobgraph jobs, and the ones that change can be personalized here too

Comment (llunaCreixent) [reply ansible/roles/graph-protocol/files/values-prd.yaml:9]: We are adding the configuration with graph-protocol, thus we add them in the yamls

**Review by llunaCreixent — APPROVED** [review]
Beautiful!!

**Pull request merged**